bin/install

```diff
@@ -12,29 +12,15 @@ get_arch() {
 }
 
 get_cpu() {
-    local machine_hardware_name
-    local cpu_type
-    machine_hardware_name="$(uname -m)"
-
-    case "$machine_hardware_name" in
-        'x86_64') cpu_type="amd64";;
-        'powerpc64le' | 'ppc64le') cpu_type="ppc64le";;
-        'aarch64') cpu_type="arm64";;
-        'armv7l') cpu_type="arm";;
-        *) cpu_type="$machine_hardware_name";;
-    esac
-
-    echo "$cpu_type"
```

Comment:
ByteNess's repo uses the correct system values for these so no need to replace

Reply:
With the https://github.com/ByteNess/aws-vault/releases/tag/v7.8.0 release, it looks likte this mapping is still needed.

Reply:
Updated to include this again.